spec/acceptance/cs_commit_spec.rb

```diff
@@ -113,13 +113,18 @@ class { 'corosync':
   end
 
   it 'creates the cib and a shadow cib' do
-    shell('cibadmin --query')
-    shell('CIB_shadow=puppet cibadmin --query')
+    if fact('osfamily') == 'RedHat'
+      shell('pcs cluster cib')
+      shell('pcs cluster cib -f puppet')
```

Comment:
what does this do? create file in current dir?

Reply:
Yes. Maybe /var/lib/pacemaker/cib/shadow.puppet would be the most logical path?

Reply:
or File.join(Puppet[:vardir], "shadow.${name}")